### PR TITLE
[WIP] docs(pkg): readme file for add package

### PR DESCRIPTION
**Pull request by ematipico**
**What kind of change does this PR introduce?**
Documentation. Part of #247

**Did you add tests for your changes?**
No
**If relevant, did you update the documentation?**
yes
**Summary**
Created read me file for **add** sub package

**Does this PR introduce a breaking change?**
No

### CODE REVIEW

File: packages/add/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+```bash
+npm i -D webpack-cli @webpack-cli/add
+```
+
+## Usage
+
+```bash
+webpack add
+```
```
Comment (ev1stensberg) [inline packages/add/README.md]: Maybe a screenshot would be good, as well as using it programatically. Could you do the same for the info pkg?

Comment (ematipico) [reply packages/add/README.md]: Yes! I also wonder if there's need of more information inside the description?

Comment (ev1stensberg) [reply packages/add/README.md]: Some things like a screenshot, what we're asking for etc.. would be nice

Comment (ev1stensberg) [reply packages/add/README.md]: Any update on this?

**Comment by ev1stensberg** [comment]
@ematipico any update?

**Pull request merged**